pkg/lib/facade/coordinator.go

```diff
@@ -1361,7 +1364,24 @@ func (c *Coordinator) MFAListRecoveryCodes(userID string) ([]*mfa.RecoveryCode,
 }
 
 func (c *Coordinator) GetUsersByStandardAttribute(attributeName string, attributeValue string) ([]string, error) {
-	claims, err := c.Identities.ListByClaim(attributeName, attributeValue)
+	var loginIDKeyType model.LoginIDKeyType
+	switch attributeName {
+	case stdattrs.Email:
+		loginIDKeyType = model.LoginIDKeyTypeEmail
+	case stdattrs.PhoneNumber:
+		loginIDKeyType = model.LoginIDKeyTypePhone
+	case stdattrs.PreferredUsername:
+		loginIDKeyType = model.LoginIDKeyTypeUsername
+	default:
+		return nil, fmt.Errorf("attributeName must be email, phone_number or preferred_username.")
```

Comment:
`return nil, api.ErrGetUsersInvalidArgument.New("attributeName must be email, phone_number or preferred_username.")`

We should do this here because it is the place where we know attributeName is invalid.

Reply:
moved api invalid error inside coordinator